keyboards/4pplet/waffling60/rev_e/keymaps/via/rules.mk

```
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Required for conditional encoder_map_enable code in `keymap.c` to trigger
```suggestion
VIA_ENABLE = yes
ENCODER_MAP_ENABLE = yes
```